README.md

```diff
@@ -10,13 +10,27 @@ This image allows you to run collectd in a completelly containerized
 environment, but while retaining the ability to report statistics about the
 _host_ the collectd container is running on.
 
+The Signal Fx collectd Docker plugin is also included, and will report metrics
+about your other containers.
```

Comment:
It will also report metrics about the collectd container itself, so technically "about your containers" is more accurate.